### DISCUSSION TITLE
How can I get all fields' value instead of $get('field_name').

### DISCUSSION THREAD

**Discussion by NaungYeHtet** [filamentphp/filament#9401] [Help]
How can I get values from all field like $request->all(), I got only single value using $get('field_name').

I am trying to make a factory model base on the form values with saving the data to database.

    Action::make('preview')
                ->icon('heroicon-o-eye')
                ->color('gray')
                ->modalContent(function (Request $request, Get $get): View {
                    $record = MyModel::factory()->make([
                        'field_1' => $get('field_1'),
                        'field_2' => $get('field_2'),
                    ]);
                    return view(
                        'website.preview', [
                            'record' => $record,
                        ],
                    );
                })
                ->modalSubmitAction(false)
                ->modalCancelAction(false)
                ->slideOver(),

**Comment by thalisvilela** [comment]
Hello buddy!!! I was able to get using get('*')

**Reply by xibn** [reply]
@thalisvilela try using `get()` without passing any arguments, you'll get the same result but every value will have the matching field name as a key instead of the normal index. 👍

**Comment by aliveisi** [comment]
`$get()`
